pkg/apis/policy/types.go

```diff
@@ -215,6 +208,12 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume
+	// AllowedUnsafeSysctls is a list of explicitly allowed unsafe sysctls, defaults to none, * means all
+	// +optional
+	AllowedUnsafeSysctls []string
+	// ForbiddenSysctls is a list of explicitly forbidden sysctls, defaults to none, * means all
```

Comment:
how do both list relate? This must be defined (also compare @liggitt's validation questions).

Reply:
https://github.com/kubernetes/kubernetes/pull/63717/files#r192786580